iceoryx_posh/include/iceoryx_posh/internal/roudi/port_manager.hpp

```diff
@@ -120,9 +121,10 @@ class PortManager
     void addEntryToServiceRegistry(const capro::IdString_t& service, const capro::IdString_t& instance) noexcept;
     void removeEntryFromServiceRegistry(const capro::IdString_t& service, const capro::IdString_t& instance) noexcept;
 
+    // clang-format off
```

Comment:
Maybe one word why i did this:
Problem is that noexcept is put into a new line because of the 120 char limit. When i try to reduce it with `using ServiceDescription = capro::ServiceDescription` then i solve it here but have the same problem with the function below.

Reply:
which version of clang-format are you using? I get the following
```
    template <typename T, std::enable_if_t<std::is_same<T, iox::build::OneToManyPolicy>::value>* = nullptr>
    cxx::optional<ProcessName_t>
    doesViolateCommunicationPolicy(const capro::ServiceDescription& service) const noexcept;
```
Essentially what it was before

Reply:
sorry mistake on my side